1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,67 @@
 // Write your code here
+let pics= document.querySelectorAll("img");
+let btnAutoback = document.getElementById("auto-back");
+let btnBack     = document.getElementById("back");
+let btnStop     = document.getElementById("stop");
+let btnForward  = document.getElementById("forward");
+let btnAutoforward  = document.getElementById("auto-forward");
+let counter     = 0;
+let forwardInterval;
+let backwardInterval;
+
+pics.forEach((img, i) => {
+img.style.left = `${i*100}%`;
+});
```

Comment:
Seems to work - the image is hidden by putting it "off screen".  But what if there are hundreds of images.  Might be better to have an array of image urls and work with that.